libraries/chain/finality/finalizer.cpp

```diff
@@ -242,15 +248,13 @@ my_finalizers_t::fsi_map my_finalizers_t::load_finalizer_safety_info() {
       // ------------------------------------------------------------------------------------------------
       uint64_t magic = 0;
       fc::raw::unpack(persist_file, magic);
-      EOS_ASSERT(magic == fsi_t::magic_unversioned || magic == fsi_t::magic, finalizer_safety_exception,
+      EOS_ASSERT(magic == fsi_t::magic, finalizer_safety_exception,
                  "bad magic number in finalizer safety persistence file: ${p}", ("p", persist_file_path));
 
-      // If we expect the file to contain a version number, read it. We can load files with older
-      // versions, but it better not be a file with a version higher that the running nodeos understands.
-      // ------------------------------------------------------------------------------------------------
+      // We can load files with older, but not files with a version higher that the running nodeos understands.
+      // -----------------------------------------------------------------------------------------------------
       uint64_t file_version = 0; // default version for file with magic == fsi_t::magic_unversioned
```

Comment:
Change this comment now that `magic_unversioned` has been removed.

Reply:
done

Reply:
Did you push it?

Reply:
I think you changed another comment I didn't notice earlier in the code, but not yet the one on this line.

Reply:
oops, sorry, I just fixed the other one.